uaclient/status.py

```diff
@@ -191,6 +191,10 @@ class UserFacingStatus(enum.Enum):
 MESSAGE_INVALID_SERVICE_OP_FAILURE_TMPL = """\
 Cannot {operation} '{name}'
 For a list of services see: sudo ua status"""
+MESSAGE_UNEXPECTED_ERROR_DURING_OP_TMPL = """\
+Unexpected error(s) occurred during {operation}
+For more details, see the log: /var/log/ubuntu-advantage.log
+To file a bug run: ubuntu-bug ubuntu-advantage-tools"""
```

Comment:
I know I suggested the change, but did we confirm that this is how we want to do bug reporting? @powersj, what do you think, `ubuntu-bug` or direct link?

Reply:
I'm fine with ubuntu-bug